custom_components/powercalc/data/lidl/HG06492C/model.json

```
@@ -0,0 +1,14 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Gosund SP1 Tasmota",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.3.3:docker"
+    },
+    "name": "HG06492C",
```

This must be the full light model name, which will also be represented in supported models listing.

```suggestion
    "name": "Livarno Lux E27 bulb CCT",
```